crates/snapbox/src/data.rs

```diff
@@ -117,13 +187,17 @@ impl Data {
         match &self.inner {
             DataInner::Binary(_) => None,
             DataInner::Text(data) => Some(data.to_owned()),
+            #[cfg(feature = "json")]
+            DataInner::Json(value) => Some(serde_json::to_string_pretty(value).unwrap()),
         }
     }
 
     pub fn to_bytes(&self) -> Vec<u8> {
         match &self.inner {
             DataInner::Binary(data) => data.clone(),
             DataInner::Text(data) => data.clone().into_bytes(),
+            #[cfg(feature = "json")]
+            DataInner::Json(value) => serde_json::to_vec(value).unwrap(),
```

Comment:
Is this doing the right thing?

Reply:
I believe that it is doing the correct thing, [`to_vec`](https://docs.rs/serde_json/1.0.85/serde_json/ser/fn.to_vec.html) returns a `Vec<u8>`

Reply:
Is it pretty or basic?  Which should we prefer?